cypress/README.md

```diff
@@ -0,0 +1,16 @@
+# install all dependencies of a project, e.g. Cypress
```

Comment:
Why not to add to common readme file?

Reply:
actually it was made specifically for DevOps, maybe move to common readme